Homework.java

```diff
@@ -2,19 +2,29 @@
 
 import org.junit.Test;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 public class Homework {
 
-    private static int secondLargestNumber(int[] arr) {
+    private static Integer secondLargestNumber(List<Integer> arr) {
 
-        int maxVal = 0;
-        int secondMaxVal = 0;
+        Integer maxVal = Integer.MIN_VALUE;
```

Comment:
Bardzo ładnie.

A co byś zrobił, gdybyś nie miał takich fajnych literałów? Na przykład gdybyś miał wybrać drugi największy elelment z listy obiektów jakiejś klasy? Nie masz literałów, ale oczywiście istnieje jakiś komparator, skoro używamy słowa "większy" (nie chodzi mi o to, żebyś tu użył komparatora dla intów, ale jak można zainicjować maxVal i tą drugą... :)